src/Aspire.Dashboard/Model/ResourceIconHelpers.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Dashboard.Model;
+
+using Microsoft.FluentUI.AspNetCore.Components;
+using Icons = Microsoft.FluentUI.AspNetCore.Components.Icons;
+
+internal static class ResourceIconHelpers
+{
+    /// <summary>
+    /// Maps a resource to a default icon.
+    /// </summary>
+    public static Icon GetIconForResource(ResourceViewModel resource)
```

Comment:
These icons were all static before now it's creating them up per call.

Reply:
Yeah because I couldn't find a better way to do colors. If someone has a better way I am all ears. Color is an instance property. Maybe some CSS selector would work and I can spend some time looking for one if we are worried about it.